pkg/planner/core/physical_plan_test.go

```diff
@@ -437,3 +437,57 @@ func TestPhysicalPlanMemoryTrace(t *testing.T) {
 	pp.MPPPartitionCols = append(pp.MPPPartitionCols, &property.MPPPartitionColumn{})
 	require.Greater(t, pp.MemoryUsage(), size)
 }
+
+func TestPhysicalTableScanExtractCorrelatedCols(t *testing.T) {
```

Comment:
Can we just use json output file to compare the whole physical plan? It is hard to see what is the purpose of this test case.

Reply:
I can add some comments.

It is only used to test `ExtractCorrelatedCols`, compare the whole plan do not meet our needs.